images/access-setup/Dockerfile

```diff
@@ -22,4 +23,4 @@ USER 65532:65532
 ENV WORK_DIR /workspace
 VOLUME /workspace
 WORKDIR /workspace
-CMD ["/opt/access-setup/bin/setup_compute.sh"]
+CMD ["/opt/access-setup/bin/setup_compute.sh"]
```

Comment:
There is no change here except the newline has been removed, is this an intentional change? IMO the file should end with a newline.

Reply:
you are right, fixed